### PR TITLE
xGov-11 - YourPlace -> To withdraw

### PR DESCRIPTION
**Pull request by 0x9090**
YourPlace is an open source, decentralized, social media protocol built on IPFS and the Algorand blockchain. This proposal is to help us hire a few junior devs and fund the infrastructure needed to grow us beyond the indie developer shop that we currently are.

### CODE REVIEW

File: Proposals/xgov-10.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+period: 10
+title: YourPlace
+author: Austin Lawrence (nops.algo)
```
Comment (SudoWeezy) [inline Proposals/xgov-10.md:4]:
```suggestion
author: Austin Lawrence (@0x9090)
```

File: Proposals/xgov-10.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+company_name: YourPlace Inc
+category: dApp
+focus_area: Identity
+open_source: Yes
+amount_requested: 1000000
+status: Draft
+---
+
+## Abstract
+[YourPlace](https://yourplace.network) is a decentralized, uncensorable, social media protocol build on IPFS and the Algorand blockchain. This open source software was built to distribute social media posts, profiles and content.
```
Comment (SudoWeezy) [inline Proposals/xgov-10.md:14]:
```suggestion
<a href="https://yourplace.network">YourPlace</a> is a decentralized, uncensorable, social media protocol build on IPFS and the Algorand blockchain. This open source software was built to distribute social media posts, profiles and content.
```

File: Proposals/xgov-10.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+The goal of YourPlace is to give all the power and decisions back to the user, while taking away as much control from the developer as possible. This is in keeping with open source and blockchain principals, but applied to social media.
+
+We are trying to create a truly open and cryptographically enforced free-speech platform to connect people and ideas across the world. We aim for fun and freedom, with no compromise on safety.
+
+This xGov proposal is for the seed funding of YourPlace Inc. which will allow us to expand beyond our current solo-developer work stream.
+Currently, about 60% of the code for the open alpha has been developed and about 70% of the v1 [white paper](https://whitepaper.yourplace.network) has been written.
```
Comment (SudoWeezy) [inline Proposals/xgov-10.md:20]:
```suggestion
Currently, about 60% of the code for the open alpha has been developed and about 70% of the v1 <a href="https://whitepaper.yourplace.network">white paper</a> has been written.
```

File: Proposals/xgov-10.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+YourPlace Inc. is requesting 1M ALGO (~$170k @ today's valuation) to fund infrastructure and engineering resources to get the project out into the public domain. This will allow us to run for at least 1 year, and potentially longer depending on future ALGO growth.
+
+## Team
+[Nops](https://www.linkedin.com/in/0x9090) is a veteran of the software security industry and has founded engineering teams at major companies like Coinbase, AWS, SpaceX, HubSpot and IBM.
```
Comment (SudoWeezy) [inline Proposals/xgov-10.md:24]:
```suggestion
<a href="https://www.linkedin.com/in/0x9090">Nops</a> is a veteran of the software security industry and has founded engineering teams at major companies like Coinbase, AWS, SpaceX, HubSpot and IBM.
```

File: Proposals/xgov-10.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+He is a proven leader who has grown teams and secured hyper growth companies through more than one successful IPO. Nops has founded small businesses, worked for Fortune 10 companies and done a little of everything in-between.
+Nops is passionate about freedom, software and security. And now he is now looking to shake up the social media formula with his latest blockchain project, YourPlace.
+
+This xGov proposal will help to fund the contracting of two junior level engineers to work directly under Nops. Their only focus will be to help develop the YourPlace software towards the open beta in 2024.
+
+## Experience with Algorand
+Nops has been working in the software security field for over 20 years and has been in the crypto space for nearly a decade. He was an early retail investor in Algorand and a major proponent of the technology.
+Some of his open source Algorand projects include:
+* [AlgoGovernanceEstimator](https://github.com/0x9090/AlgoGovernanceEstimator)
```
Comment (SudoWeezy) [inline Proposals/xgov-10.md:33]:
```suggestion
* <a href="https://github.com/0x9090/AlgoGovernanceEstimator">AlgoGovernanceEstimator</a>
```

File: Proposals/xgov-10.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+* [AlgoAuth](https://github.com/NullableLabs/AlgoAuth)
```
Comment (SudoWeezy) [inline Proposals/xgov-10.md:34]:
```suggestion
* <a href="https://github.com/NullableLabs/AlgoAuth">AlgoAuth</a>
```

File: Proposals/xgov-10.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+## Benefits for the community
+YourPlace is a dApp which allows the public to create communities and self-publish content, with Algorand and IPFS being the hearts of the network. This gives Algorand users a digital, on-chain home to call their own. YourPlace also integrates with NFD and other Algo ecosystem providers, encouraging their usage.
+YourPlace is based on the native ALGO token. All non-fungible actions are stored as an Algo transaction, so it should drastically increase the usage of the Algorand blockchain. This increase in usage would ideally increase the price for the Algo token for all holders.
+Being an open social media protocol gives users the ability to extend and modify the protocol to deliver content and services of all types. We want to bring back the fun and profile customization from social media of old, while preserving the features and safety mechanisms you'd expect to see today.
+Freedom of speech is a cryptographically enforced principal in YourPlace and we've replaced centralized content moderation with decentralized, personal content controls.
+
+## Additional information
+* [YourPlace Beta](https://beta.yourplace.network)
```
Comment (SudoWeezy) [inline Proposals/xgov-10.md:50]:
```suggestion
* <a href="https://beta.yourplace.network">YourPlace Beta</a>
```

File: Proposals/xgov-10.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+* [YourPlace White Paper](https://whitepaper.yourplace.network) (Draft)
```
Comment (SudoWeezy) [inline Proposals/xgov-10.md:51]:
```suggestion
* <a href="https://whitepaper.yourplace.network">YourPlace White Paper</a>
```

File: Proposals/xgov-10.md [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+* [Twitter](https://twitter.com/YourPlace_Inc)
```
Comment (SudoWeezy) [inline Proposals/xgov-10.md:52]:
```suggestion
* <a href="https://twitter.com/YourPlace_Inc">Twitter</a>
```

File: Proposals/xgov-10.md [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+* [LinkedIn](https://www.linkedin.com/company/yourplace-network/)
```
Comment (SudoWeezy) [inline Proposals/xgov-10.md:53]:
```suggestion
* <a href="https://www.linkedin.com/company/yourplace-network/">LinkedIn</a>
```

**Review by SudoWeezy — CHANGES_REQUESTED** [review]
Thanks for your proposal @0x9090. I have suggested some formatting changes to have a future-proof structure.

File: Proposals/xgov-10.md (hunk 2 above)
Comment (0x9090) [reply Proposals/xgov-10.md:14]: Is the standard for xGov requests to use HTML links, instead of Markdown?

File: Proposals/xgov-11.md [hunk 11]
```diff
@@ -0,0 +1,58 @@
+---
+period: 10
+title: YourPlace
+author: Austin Lawrence (@0x9090)
+company_name: YourPlace Inc
+category: dApps
+focus_area: Identity
+open_source: Yes
+amount_requested: 1000000
```
Comment (SudoWeezy) [inline Proposals/xgov-11.md:9]: If your project requires multiple milestones, split them into multiple proposals see: https://github.com/algorandfoundation/ARCs/pull/209

**Review by SudoWeezy — CHANGES_REQUESTED** [review]

**Comment by SudoWeezy** [comment]
Hello.
Please update your proposal to match the template provided here. 
[https://github.com/algorandfoundation/ARCs/blob/main/assets/arc-0034/TemplateForm.md](https://github.com/algorandfoundation/ARCs/blob/main/assets/arc-0034/TemplateForm.md)
Thanks.

**Pull request closed without merging**